### PR TITLE
Fix async url

### PR DESCRIPTION
**Pull request by dimakis**
The URL which is built by the SDK has a small bug. 
As the parameter `async` is a bool --> `True` the URL gets built as such:
`https://api.openshift.com//api​/kafkas_mgmt​/v1​/kafkas?async=True`
the correct URL should be:
`https://api.openshift.com//api​/kafkas_mgmt​/v1​/kafkas?async=true`

notice the subtle difference in the capitalisation of `True`.

this template edit fixes this.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "configurations": [
+        {
+            "name": "Dynamic Python: File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "justMyCode": false,
+            "env": {
+                "OFFLINE_TOKEN": "YOUR OFFLINE TOKEN HERE"
```
Comment (wtrocki) [inline .vscode/launch.json:10]: Process would also have access to system env. Thus this is going to override it and be generally less useful than no env defined
```suggestion
```

File: auth/rhoas_auth.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from keycloak import KeycloakOpenID
+from os import environ;
+
+def get_access_token():
+    offlineToken = environ.get('OFFLINE_TOKEN')
+    # Configure client
+    keycloak_openid = KeycloakOpenID(server_url="https://sso.redhat.com/auth/",
```
Comment (wtrocki) [inline auth/rhoas_auth.py:7]: Those should be defined as constants. 
 get_access_token() should allow to override those values.

File: scripts/templates/rest.mustache [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+                timeout = urllib3.Timeout(
+                    connect=_request_timeout[0], read=_request_timeout[1])
+
+        try:
+            # For `POST`, `PUT`, `PATCH`, `OPTIONS`, `DELETE`
+            if method in ['POST', 'PUT', 'PATCH', 'OPTIONS', 'DELETE']:
+                # Only set a default Content-Type for POST, PUT, PATCH and OPTIONS requests
+                if (method != 'DELETE') and ('Content-Type' not in headers):
+                    headers['Content-Type'] = 'application/json'
+                if query_params[0] == ('async', True):
```
[the remark this answers is not in the document]
Comment (dimakis) [reply scripts/templates/rest.mustache:141]: I don't see any major reason why this shouldn't be the final version? if `query_params[0]` does not match that tuple then the method will continue as before. 
maybe I'm missing something? 🤔

**Review by wtrocki — APPROVED** [review]

**Review by wtrocki — APPROVED** [review]
Good as temporary solution.

File: auth/rhoas_auth.py [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 
 def get_access_token():
     offlineToken = environ.get('OFFLINE_TOKEN')
     # Configure client
-    keycloak_openid = KeycloakOpenID(server_url="https://sso.redhat.com/auth/",
-                                 client_id="cloud-services",
-                                 realm_name="redhat-external")
+    keycloak_openid = KeycloakOpenID(server_url=DEFAULT_AUTH_URL,
+                                 client_id=DEFAULT_CLIENT_ID,
+                                 realm_name=REALM_NAME)
```
Comment (dimakis) [inline auth/rhoas_auth.py:10]: I am on the VPN but even so, when I run an example, I get this error:
```python
keycloak.exceptions.KeycloakPostError: 404: b'{"error":"Realm does not exist"}'
```

@wtrocki, have you seen this before?

Comment (wtrocki) [reply auth/rhoas_auth.py:10]: You have 2 PRs. Should other one be closed?

With config like this i do not see errors:
```
    keycloak_openid = KeycloakOpenID(server_url="https://sso.redhat.com/auth/",
                                 client_id="cloud-services",
                                 realm_name="redhat-external")
                                 
                             ```

**Pull request merged**